packages/grafana-runtime/src/services/index.ts

```diff
@@ -8,14 +8,4 @@ export * from './legacyAngularInjector';
 export * from './live';
 export * from './LocationService';
 export * from './appEvents';
-export {
-  type PluginExtensionRegistry,
-  type PluginExtensionRegistryItem,
-  setPluginsExtensionRegistry,
-} from './pluginExtensions/registry';
-export {
-  type PluginExtensionsOptions,
-  type PluginExtensionsResult,
-  getPluginExtensions,
-} from './pluginExtensions/extensions';
-export { type PluginExtensionPanelContext } from './pluginExtensions/contexts';
+export * from './pluginExtensions';
```

Comment:
I think it would be nice to stick to named exports so we can easily identify what we're exposing.

Reply:
+1